src/Autonomous.cpp

```diff
@@ -269,9 +279,16 @@ void Autonomous::autonomyIter()
 			}
 		}
 
-		while (viz_window.pollWindowEvent() != -1) {}
-		viz_window.drawPoints(transformReadings(lidar_scan, VIZ_BASE_TF), sf::Color::Red, 3);
-		drawPose(pose, pose, sf::Color::Black);
+		// Send lidar points to visualization
+		const points_t lidar_scan_transformed = transformReadings(lidar_scan, VIZ_BASE_TF);
+		for (point_t point : lidar_scan_transformed)
+		{
+			publish(point, lidar_pub);
```

Comment:
I've actually never used ROS2 before, but in ROS 1 there are different message types that you can use to publish e.g. a list of points, rather than a single point at a time. We might eventually want to switch to use those---maybe it would be more computationally efficient, or play better with other ROS tools like Rviz.